aspnetcore/security/app-secrets.md

```diff
@@ -18,7 +18,9 @@ By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Kirk Larkin](https://tw
 
 [View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/security/app-secrets/samples) ([how to download](xref:index#how-to-download-a-sample))
 
-This document explains how to manage sensitive data for an ASP.NET Core app on a development machine. Never store passwords or other sensitive data in source code. Production secrets shouldn't be used for development or test. Secrets shouldn't be deployed with the app. Instead, production secrets should be accessed through a controlled means like environment variables or Azure Key Vault. You can store and protect Azure test and production secrets with the [Azure Key Vault configuration provider](xref:security/key-vault-configuration).
+This document explains how to manage sensitive data for an ASP.NET Core app on a development machine. Never store passwords or other sensitive data in source code or configuration files. Production secrets shouldn't be used for development or test. Secrets shouldn't be deployed with the app.  Production secrets should be accessed through a controlled means like Azure Key Vault. Azure test and production secrets can be stored and protected with the [Azure Key Vault configuration provider](xref:security/key-vault-configuration).
+
+[!INCLUDE [managed-identities-test-non-production](~/includes/managed-identities-test-non-production.md)]
```

Comment:
This include file isn't a perfect fit here. It begins "This article uses a local database that doesn't require the user to be authenticated." That was written for a tutorial. This article uses a few database examples, including one that does require authentication. Consider using revised include file text in a variant of the include file: "Some of the connection strings shown in this article are for local databases that don't require the user to be authenticated."
>